gui/ProjectVentilatorGUI.pro

```diff
@@ -1,14 +1,25 @@
-QT += core quick charts
+QT += core quick charts serialport
 CONFIG += c++17
 DEFINES += QT_DEPRECATED_WARNINGS
 
-SOURCES += $$files("*.cpp")
+SOURCES += $$files("*.cpp") \
+    ../common/generated_libs/network_protocol/network_protocol.pb.c \
+    ../common/third_party/nanopb/pb_common.c \
+    ../common/third_party/nanopb/pb_decode.c \
+    ../common/third_party/nanopb/pb_encode.c
 SOURCES += $$files("$$PWD/../common/**/*.c")
-HEADERS += $$files("*.h")
+HEADERS += $$files("*.h") \
+    ../common/generated_libs/network_protocol/network_protocol.pb.h \
+    ../common/third_party/nanopb/pb.h \
+    ../common/third_party/nanopb/pb_common.h \
+    ../common/third_party/nanopb/pb_decode.h \
+    ../common/third_party/nanopb/pb_encode.h
 HEADERS += $$files("$$PWD/../common/**/*.h")
 INCLUDEPATH += $$PWD/../common/third_party/nanopb
 RESOURCES += qml.qrc Logo.png
-DISTFILES += Logo.png
+DISTFILES += Logo.png \
+    ../common/generated_libs/network_protocol/network_protocol.proto \
```

Comment:
I don't think these files have to be in DISTFILES - they aren't involved in building the GUI at all.